### ISSUE TITLE
marketplace app pages styling look ugly.

### ISSUE THREAD

**Issue by the-j0k3r** [StylishThemes/GitHub-Dark#699]
* **Browser**: FF 66.0.1
* **Operating System**: Win10pro/Linux 
* **Screenshot**:

![marketplace](https://user-images.githubusercontent.com/31389848/42932352-f8589da8-8b39-11e8-993a-05ad65b85aca.gif)

e.g https://github.com/marketplace/buddy

- [x] font color on the right list is not really white `filter-item.selected` and friends aka `h4` and `.f3` are not styled?
- [x] the top border on the right box background is being overridden by `.bg-blue:not(.text-bold)`

* **HTML of the section where the issue occurs**:

````html
<li>
      <a href="/marketplace/buddy/plan/MDIyOk1hcmtldHBsYWNlTGlzdGluZ1BsYW45MDM=#pricing-and-setup" class="filter-item ws-normal d-flex flex-justify-between p-2 p-md-3 mb-0 selected" data-pjax="" data-pjax-preserve-scroll="">
        
<div class="flex-auto flex-self-center">
  <h4 class="f3 d-inline-block marketplace-plan-emphasis">Free
  </h4>
  <p class="text-small mb-0 pr-2">1x concurrency, 1 GB RAM </p>
</div>
  <span class="f3 float-right marketplace-plan-emphasis">$0</span>

      </a>
    </li>
````

````html
<div class="position-absolute top-0 left-0 width-full bg-blue pt-1"></div>
````

**Comment by the-j0k3r** [comment]
After some more investigation issue nr 2 rests with @silverwind  re:  fix to issue #674 https://github.com/StylishThemes/GitHub-Dark/commit/093721583fe1cafc79d71146d641a4e13b4e92a2

**Comment by the-j0k3r** [comment]
A fix proposal is up.
